src/components/atoms/SearchedLocationDisplay/index.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import useRegionQueries from "@/hooks/useRegionQueries";
+import { PageSearchParams } from "@/types/pageSearchParams";
+import { MdLocationOn } from "react-icons/md";
+
+function SearchedLocationDisplay({ searchParams }: PageSearchParams) {
+  const cityId = searchParams?.cityId ? parseInt(searchParams?.cityId, 10) || 0 : 0;
+  const countryId = searchParams?.countryId ? parseInt(searchParams?.countryId, 10) || 0 : 0;
+  const districtId = searchParams?.districtId ? parseInt(searchParams?.districtId, 10) || 0 : 0;
```

Comment:
parseInt의 두번째 인자는 몇진수로 나타낼 것이냐를 결정하는 인자예요. default가 10이기 때문에 제거를 해주셔도 되고
이렇게 구문을 수정할 수 있을 것 같아요

```suggestion
  const cityId = parseInt(searchParams?.cityId || 0, 10);
  const countryId = parseInt(searchParams?.countryId || 0, 10);
  const districtId = parseInt(searchParams?.districtId || 0, 10);
```

Reply:
parseInt의 두 번째 인자는 넣지 않으면 ESLint 워닝이 발생해서 집어넣었습니다. 구문은 로직에 맞게 수정했습니다.

Reply:
parseInt에 대해 더 알아봤는데 기본값이 10으로 볼 수는 없고 0x로 시작하는 케이스에 16진수로 해석하게 되네요
그래서 이대로 유지하시거나 parseDecimal같은 유틸로 만들어서 반복적으로 10을 넣어줘야 하는 것을 해소할 수도 있겠습니다!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/parseInt
https://velog.io/@katherine_12/Javascript-ESLint%EC%97%90%EC%84%9C-parseIntstring%EC%9D%80-%EC%99%9C-%EC%97%90%EB%9F%AC%EA%B0%80-%EB%82%A0%EA%B9%8C